drivers/display/display_sdl.c

```diff
@@ -116,14 +117,13 @@ static void sdl_task_thread(void *p1, void *p2, void *p3)
 		sdl_display_zoom_pct = CONFIG_SDL_DISPLAY_ZOOM_PCT;
 	}
 
-	int rc = sdl_display_init_bottom(config->height, config->width, sdl_display_zoom_pct,
-					 use_accelerator, &disp_data->window, dev,
-					 &disp_data->renderer, &disp_data->mutex,
-					 &disp_data->texture, &disp_data->read_texture,
-					 &disp_data->background_texture,
-					 CONFIG_SDL_DISPLAY_TRANSPARENCY_GRID_CELL_COLOR_1,
-					 CONFIG_SDL_DISPLAY_TRANSPARENCY_GRID_CELL_COLOR_2,
-					 CONFIG_SDL_DISPLAY_TRANSPARENCY_GRID_CELL_SIZE);
+	int rc = sdl_display_init_bottom(
+		config->height, config->width, sdl_display_zoom_pct, use_accelerator,
+		&disp_data->window, dev, &disp_data->renderer, &disp_data->mutex,
+		&disp_data->texture, &disp_data->read_texture, &disp_data->background_texture,
+		CONFIG_SDL_DISPLAY_TRANSPARENCY_GRID_CELL_COLOR_1,
+		CONFIG_SDL_DISPLAY_TRANSPARENCY_GRID_CELL_COLOR_2,
+		CONFIG_SDL_DISPLAY_TRANSPARENCY_GRID_CELL_SIZE, config->label);
```

Comment:
Regarding argument ordering, I'd put this further up in the list perhabs after  `&disp_data->window, dev,`